dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -127,3 +127,17 @@ startsecs=10
 dependent_startup=true
 dependent_startup_wait_for=start:exited
 {% endif %}
+
+{% if not skip_pcied %}
```

Comment:
Are there any platforms for which we would *not* want to run this daemon?

Reply:
if the configuration file doesn't exist under platform directory, then this daemon will exit as expected and won't run again.

Reply:
I understand. I just wonder if there would be any platforms for which we wouldn't want to monitor the PCIe devices.